README.md

```diff
@@ -9,6 +9,8 @@ Sample projects showing how to integrate Blockly into your project.
 - [``blockly-react-sample``](blockly-react/README.md): Blockly in a React project, defines a React Blockly Component.
 - [``blockly-svelte-sample``](blockly-svelte/README.md): Blockly in a Svelte project, defines a Svelte Blockly Component.
 - [``blockly-vue-sample``](blockly-vue/README.md): Blockly in a Vue project, defines a Vue Blockly Component.
+- [``blockly-rtc-sample``](blockly-rtc/README.md): Realtime collaboration environment on top of the Blockly framework.
```

Comment:
I don't think this belongs in this section. The readme reads "samples showing how to integrate Blockly into your project".

We should create a separate list for listing these kinds of samples.

Reply:
Would the rendering walkthrough sample go under the "samples showing how to integrate Blockly into your project" heading or a new one with the rtc sample?